pkg/dnsprovider/providers/google/clouddns/clouddns_test.go

```diff
@@ -27,19 +27,19 @@ import (
 	"k8s.io/federation/pkg/dnsprovider/tests"
 )
 
-func newTestInterface() (dnsprovider.Interface, error) {
+func newTestInterface(zones []string) (dnsprovider.Interface, error) {
 	// Use this to test the real cloud service - insert appropriate project-id.  Default token source will be used.  See
 	// https://github.com/golang/oauth2/blob/master/google/default.go for details.
 	// return dnsprovider.GetDnsProvider(ProviderName, strings.NewReader("\n[global]\nproject-id = federation0-cluster00"))
-	return NewFakeInterface() // Use this to stub out the entire cloud service
+	return NewFakeInterface(zones) // Use this to stub out the entire cloud service
 }
 
 var interface_ dnsprovider.Interface
 
 func TestMain(m *testing.M) {
 	flag.Parse()
 	var err error
-	interface_, err = newTestInterface()
+	interface_, err = newTestInterface([]string{"example.com"})
```

Comment:
Isn't the intention of making  NewFakeInterface() handle  multiple zones to test out passing multiple zones??